addons/survey/static/tests/tours/survey.js

```diff
@@ -76,4 +75,53 @@ registry.category("web_tour.tours").add('test_survey', {
         content: 'Thank you',
         trigger: 'h1:contains("Thank you!")',
     }
-]});
+];
+
+const survey_multilang_steps = [
+    {
+        content: "Select French",
+        trigger: "select[name='lang_code']",
+        run() {
+            const langSelect = document.querySelector("select[name='lang_code']");
+            langSelect.value = "fr_BE";
+            langSelect.dispatchEvent(new Event("change", { bubbles: true }));
+        },
+    },
+    {
+        content: "Check French translation",
```

Comment:
Shouldn't we instead/also translate one question and switch the step for that question, so that we check that we correctly "remember" the selected language throughout the "session"?